shared/provision/username-or-email/container.tsx

```diff
@@ -54,5 +54,5 @@ const dispatchToProps = (dispatch: Container.TypedDispatch) => ({
 
 export default Container.compose(
   Container.connect(mapStateToProps, dispatchToProps, (s, d, _: OwnProps) => ({...s, ...d})),
-  Container.safeSubmit(['onBack', 'onSubmit'], ['error', 'inlineError'])
+  Container.safeSubmit(['onBack', 'onSubmit'], ['error', 'inlineError', 'inlineSignUpLink'])
```

Comment:
Separate bug; the login button stopped working if you entered a nonexistent username.